variables.tf

```
@@ -18,7 +18,7 @@ variable "additional_thumbprints" {
   type        = list(string)
 
   validation {
-    condition     = length(var.additional_thumbprints) <= 4
+    condition     = var.additional_thumbprints != null ? length(var.additional_thumbprints) <= 4 : true
```

Good PR!!
I think this would make it more readable, what do you think?
and ... for example `!= null` will result in a different error, since length is processed even if a bool value is passed.

Of course it is OK to make errors, that is OK.
However, since the default value is null, I thought it would be better to use null as the comparison target.

```suggestion
    condition     = var.additional_thumbprints == null ? true : length(var.additional_thumbprints) <= 4
```